sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -918,10 +918,10 @@ inline pi_result piDeviceRetain(pi_device Device) {
 }
 
 inline pi_result piDeviceRelease(pi_device Device) {
-  PI_ASSERT(Device, PI_ERROR_INVALID_DEVICE);
-
-  auto UrDevice = reinterpret_cast<ur_device_handle_t>(Device);
-  HANDLE_ERRORS(urDeviceRelease(UrDevice));
+  if (Device) {

```

Comment:
I would prefer not to touch UR common code on this patch, as here we are trying to fix an error that is specific to L0. so checking that Device is not null.  Of course, it is weird that we checking here that device is not null, while in other interfaces a PI_ASSERT  is used.